sklearn/covariance/graph_lasso_.py

```diff
@@ -79,8 +79,9 @@ def alpha_max(emp_cov):
 # The g-lasso algorithm
 
 def graph_lasso(emp_cov, alpha, cov_init=None, mode='cd', tol=1e-4,
-                max_iter=100, verbose=False, return_costs=False,
-                eps=np.finfo(np.float).eps, return_n_iter=False):
+                enet_tol=1e-4, max_iter=100, verbose=False,
+                return_costs=False, eps=np.finfo(np.float).eps,
```

Comment:
It's better to be explicit with the precision: `eps=np.finfo(np.float64).eps`. We don't want platform specific behaviors.